device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json

```diff
@@ -0,0 +1,176 @@
+{
+    "chassis": {
+        "name": "7800R3A-36DM2-LC",
+        "components": [],
```

Comment:
Components are seemingly missing here which will lead to some sonic-mgmt tests to fail.
You will need these 3:
```
Aboot()
Scd(addr=0000:00:18.7)
Scd(addr=0000:08:00.0)
```

Reply:
`test_components` were skipped because your API returns 0 number of components.

- Would you want to make the API to return 3?
- is the quote needed in components names?

Reply:
It's actually the same reply as the one for the supervisor.
We only report components on the module object and no longer on the chassis.
Please ignore my comment.